bigbluebutton-html5/public/locales/en.json

```diff
@@ -250,8 +250,15 @@
     "app.presentationUploader.sent": "Sent",
     "app.presentationUploader.exportingTimeout": "The export is taking too long...",
     "app.presentationUploader.export": "Send to chat",
+    "app.presentationUploader.exportAnnotatedPresentation": "Send out a download link for the presentation with annotations included",
```

Comment:
What do you think about adding "Enable/Disable download _of the_ original presentation"?